src/main/java/org/broadinstitute/hellbender/tools/funcotator/DataSourceFuncotationFactory.java

```diff
@@ -39,6 +40,12 @@
      */
     protected Map<String, String> annotationOverrideMap;
 
+    /**
+     * If {@code true}, the backing data behind this {@link DataSourceFuncotationFactory} is based on the b37 reference AND we are using hg19 data.
```

Comment:
The FuncotationFactory should have no idea what the input variants are.  It can support annotating hg19 if backed by b37.  Is that what is happening here.  The comment is a bit confusing to me.  Should this be an enumeration?  hg19 or b37?

Reply:
No - this is a flag from the config files that will enable a b37 data source to annotate hg19/b37 variants.  It's the converse of what we have had previously (b37 variants with hg19 data sources).

I'm using this because the gnomAD hg19 version that is public is _actually_ b37 and I don't want to make a copy of it if I can help it.

This is a special case that philosophically shouldn't exist at all - we should have completely separate data sources for each reference.

However, this is faster, easier, and less confusing for users at the Broad in particular (due to our b37/hg19 false equivalence).

I'll change the comment to be a little more informative.